src/test/java/org/candlepin/test/TestUtil.java

```diff
@@ -324,13 +222,123 @@ public static String randomString() {
         return randomString(null);
     }
 
-    public static Content createContent(String id, String name) {
-        Content content = new Content(id, name, "test-type", "test-label", "test-vendor");
-        content.setContentUrl("https://test.url.com");
-        content.setGpgUrl("https://gpg.test.url.com");
-        content.setArches("x86");
+    public static Owner createOwner(String key, String displayName) {
+        return new Owner()
+            .setId(key + "-id")
+            .setKey(key)
+            .setDisplayName(displayName);
+    }
 
-        return content;
+    public static Owner createOwner(String key) {
+        return createOwner(key, key);
+    }
+
+    public static Owner createOwner() {
+        return createOwner("Test Owner " + randomInt());
+    }
+
+    public static Consumer createConsumer(ConsumerType type, Owner owner, String username) {
+        return new Consumer()
+            .setUuid(Util.generateUUID())
+            .setName("TestConsumer" + randomInt())
+            .setUsername(username)
+            .setOwner(owner)
+            .setType(type);
+    }
+
+    public static Consumer createConsumer(ConsumerType type, Owner owner) {
+        return createConsumer(type, owner, "User");
+    }
+
+    /**
+     * Create a consumer with a new owner
+     *
+     * @return Consumer
+     */
+    public static Consumer createConsumer(Owner owner) {
+        Consumer consumer = createConsumer(createConsumerType(), owner);
+
+        // TODO: FIXME: These *should* live on the main createConsumer for consistency and sanity,
+        // but for whatever reason we decided rewriting the same thing 9001 with slight variations
+        // would be more spicy. Move these if they don't break too many tests.
```

Comment:
is the '9001' a mistake in this comment?

Reply:
No, but the word "times" is missing. Basically, I was referencing that our implementation of all those methods are independent for some reason, rather than just being slight overloads that add more or less data (or chaining fluent methods).

This call in particular behaving differently wasn't something I wanted to deal with given the already huge amount of changes made elsewhere just to get things to work. At some point we should remove this method and have it be largely passthrough to the base util method.

Reply:
Okay, that makes sense!